app/Services/Manager.php

```diff
@@ -92,6 +92,77 @@ public function exportCSV($model, $hasReportback = false)
     }
 
     /**
+     * Build leaderboard data for a given Competition
+     *
+     * @param Competition $competition
+     * @param User $users
+     * @return array $leaderboard
+     */
+    public function getLeaderboard($competition, $users)
+    {
+        $rows = [];
+
+        // Get all users in bulk
+        $parameters['ids'] = implode(',', $users->pluck('id')->all());
```

Comment:
hmmm, you can just save this as `$userIds` or something. Only reason I saved it as `$parameters['ids']` in the code you're referencing this from is cause of how it got passed in to the API as URL params.

Here you're creating an array of an array of ids, instead of just an array of ids...